src/screens/BedScreen/BedCards.tsx

```diff
@@ -13,6 +13,11 @@ type Props = {
   navigation: GardenScreenProps<"BedsScreen">["navigation"];
 };
 
+export type LocationObj = {
+  gardenTitle: string;
+  bedTitle: string;
+} | null;
```

Comment:
Usually, avoid including the types type within the types name. Also something a bit more descriptive when it's being exported. If somethings not being exported it's kept within the scope of the current file or function you can be much less descriptive than when its exported. Also including | null would probably be best to put with where the type is being assigned. then this type can remain an 'entity' on its own, with then null its really like LocationObjProperty or something. also good to stop using null as much as you can, undefined can be used in pretty much all cases. if you refactor the null out you can just replace them all with locationTitles?: WhateverTypeNameYouGoWith

```suggestion
export type GardenTitles = {
  gardenTitle: string;
  bedTitle: string;
};
```